Missing mcap library on build

**Describe the bug:**
When trying to build the FMU, an error is raised:
```bash
/sl-5-5-osi-trace-file-player/lib/asam-osi-utilities/include/osi-utilities/tracefile/reader/MCAPTraceFileReader.h:9:10: fatal error: mcap/reader.hpp: No such file or directory
#include <mcap/reader.hpp>
```

Apparently the mcap library is expected to be installed.
This is caused by not checking out submodules recursively --> add it to the README.md

**To Reproduce:**
Steps to reproduce the behavior: Every step as described in README.md

**Describe the setup you are using:**

- Commit 9e884e2
- OS: Docker base image ubuntu:20.04

--- a/README.md
+++ b/README.md
@@ -38,9 +38,8 @@ sudo apt-get install libprotobuf-dev protobuf-compiler
 ### Clone with submodules
 
 ```bash
-git clone https://github.com/openMSL/sl-5-5-osi-trace-file-player.git
+git clone --recurse-submodules https://github.com/openMSL/sl-5-5-osi-trace-file-player.git
 cd sl-5-5-osi-trace-file-player
-git submodule update --init
 ```
 
 ### Build

--- a/README.md
+++ b/README.md
@@ -23,16 +23,22 @@ Otherwise, the FMU will return with an error.
 
 ### Dependencies
 
-Install `cmake` (at least version 3.10.2):
+Common tools for building:
 
 ```bash
-sudo apt-get install cmake
+sudo apt-get install git cmake build-essential
 ```
 
 Install `protobuf` (at least version 3.0.0):
 
 ```bash
 sudo apt-get install libprotobuf-dev protobuf-compiler
+```
+
+Install `libzstd` and `liblz4`:
+
+```bash
+sudo apt-get install libzstd-dev liblz4-dev
 ```
 
 ### Clone with submodules

